orc.js

```diff
@@ -46,28 +91,35 @@ function status() {
        status: 'modified' } ],
   untracked: [ 'orc-test.js' ] }
   */
-	r = new git.Repo('.', {}, function(err, repo) {
-		if(err) {
-			console.log('error getting repo status');
-			return;
-		}
-		repo.status(function(err, stats){
-			console.log('stats: '+JSON.stringify(stats));
-		});
-	});
+  orc.repo.status(cb);
 }
 
-app.command('checkpoint').alias('cp').description('commit all local changes and push to repo. use this all the time!').action(checkpoint);
-app.command('pull').description('pull latest from remote master into your current branch').action(pull);
-app.command('push').description("use this when you're ready to submit a pull request on github: squash your branch down to one commit, run unit tests, and push.").action(push);
-app.command('branch').description('helper for switching branches').action(branch);
-app.command('status').alias('st').description('helper for getting status').action(status); // REMOVE
+exports.main = function(dir, argv, cb) {
```

Comment:
real minor point, but I'd declare this as a function and then export it at the bottom (should be in our code style guide I guess)